tests/e2e/Count.test.ts

```diff
@@ -0,0 +1,24 @@
+import { test, expect, Page } from "@playwright/test";
+
+let page: Page;
+
+test.beforeAll(async ({ browser }) => {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  page = await browser.newPage();
+  await page.goto("/count");
+});
+
+test.afterAll(async () => {
+  await page.close();
+});
+
+test.describe("Count page", () => {
+  test("should count up from 10 to 11", async () => {
+    // Make sure the initial count is 10
+    await expect(page.locator("span")).toHaveText(["10"]);
+    // Click the button to increase the count
+    await page.locator("button", { hasText: /^\+$/ }).click();
+    // Make sure the count now has 11
+    await expect(page.locator("span")).toHaveText(["11"]);
+  });
+});
```

Comment:
A simple test has been added on the Count page.
If you are using vscode, you can also easily test with [playwright's extension](https://marketplace.visualstudio.com/items?itemName=ms-playwright.playwright).

https://user-images.githubusercontent.com/9401060/177752282-769d8fd8-9845-4613-a394-b945f46a4e1c.mp4